polaris.shopify.com/content/content/naming.mdx

```diff
@@ -606,7 +606,7 @@ When using a country as an adjective (such as when referring to currency), you m
 
 ### Similar names
 
-Being an ecommerce platform has its naming challenges. For example, a lot of feature names could include the word “shipping” or “pay”. Think merchant-first and be descriptive to differentiate the name. Picture what it’s like to have conversations with merchants about similar names like Shopify Payments and conversationsor around our pricing plans to identify names that might be confusing. confusingming a new product or feature:
+Being an ecommerce platform has its naming challenges. For example, a lot of feature names could include the word “shipping” or “pay”. Think merchant-first and be descriptive to differentiate the name. Picture what it’s like to have conversations with merchants about similar names like Shopify Payments and conversations or around our pricing plans to identify names that might be confusing a new product or feature:
```

Comment:
In re-reading this, I feel like the entire sentence is incorrect and awkward. I think we're trying to communicate more concepts than should be in one sentence but I'm not sure exactly what we're trying to say so it's tough to suggest wording. Maybe:

"Think merchant-first and be descriptive to differentiate the name. Imagine what it’s like to have conversations with merchants about similar names like Shopify Payments or around our pricing plans to identify names that might be confusing. Before naming a new product or feature:"